pkg/serviceaccount/openidmetadata.go

```diff
@@ -44,26 +44,47 @@ const (
 	JWKSPath = "/openid/v1/jwks"
 )
 
-// OpenIDMetadata contains the pre-rendered responses for OIDC discovery endpoints.
-type OpenIDMetadata struct {
-	ConfigJSON       []byte
-	PublicKeysetJSON []byte
+// OpenIDMetadataProvider returns pre-rendered responses for OIDC discovery endpoints.
```

Comment:
needs updating.

Reply:
I think the comment is accurate, right?

Reply:
the responses are pre-rendered from the perspective of the caller (it doesn't have to transform them, just dump them out to the response)